CodeMaid/Helpers/CodeItemTypeComparer.cs

```diff
@@ -139,6 +168,23 @@ private static int CalculateReadOnlyOffset(BaseCodeItem codeItem)
             return codeItemField.IsReadOnly ? 0 : 1;
         }
 
+        private static string NormalizeName(BaseCodeItem codeItem)
+        {
+            string name = codeItem.Name;
+            var interfaceItem = codeItem as IInterfaceItem;
+            if ((interfaceItem != null) && interfaceItem.IsExplicitInterfaceImplementation)
+            {
+                // Try to find where the interface ends and the method starts
+                int dot = name.LastIndexOf('.') + 1;
+                if ((dot > 0) && (dot < name.Length))
```

Comment:
A convention I've started using for 'between' checks is to put the variable in the middle, with the smaller check on the left and the larger check on the right.  Minor readability enhancement.  So for example:

`if (0 < dot && dot < name.Length)`